### ISSUE TITLE
Failed to run - Unexpected token o in JSON at position 1

### ISSUE THREAD

**Issue by stefpriols** [mifi/SimpleInstaBot#156]
**To Reproduce**
Default settings.
When I start the bot, it login into my account and get this error.

**Screenshots**

**Desktop (please complete the following information):**
 - macOS Mojave 10.14.6
 - Google Chrome
 - MacBook Pro 2015 - Google Chrome version 103.0.5060.114 
 
**Additional context**

**Provide me with HTML**

Did you receive an error while the bot is running?
```
Missing graphql in page, falling back to alternative method...
Failed to run - Unexpected token o in JSON at position 1

```
To make it easier for me to understand why it's failing, please copy and share with me the HTML of the button (or even better the whole page) and share it, by following these instructions:

- Make sure you run the command after getting the error message
- make sure that you open the dev tools **on the correct window (the instagram page)**

Use <kbd>Option</kbd> + <kbd>⌘</kbd> + <kbd>J</kbd> (on macOS), or <kbd>Shift</kbd> + <kbd>CTRL</kbd> + <kbd>J</kbd> (on Windows/Linux) to open devtools.

Using the Web Inspector (F12), go to the Elements tab, right click on the tag in your code and select Copy->Copy outerHTML. Then paste that here

`<html><head></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">for (;;);{"redirect":"https:\/\/www.instagram.com\/...\/"}</pre></body></html>`

If you feel it's private information you can instead email to simpleinstabot@yankee.no

**Comment by dankillam** [comment]
I got this too. I downloaded the latest version and it worked. I ended up getting a different error, but I've received this before. 

`Sorry, this page isn't available.
The link you followed may be broken, or the page may have been removed. [Go back to Instagram.](https://www.instagram.com/)`

When I go to the page in Firefox it does the same thing. When I go to the page in Chrome, it works.

For your issue, it seems like downloading the latest issue fixes the problem.
